sdcm/cluster_aws.py

```diff
@@ -437,6 +437,7 @@ def __init__(self, ec2_instance, ec2_service, credentials, parent_cluster,  # py
                                       node_prefix=node_prefix,
                                       dc_idx=dc_idx)
         if not cluster.Setup.REUSE_CLUSTER:
+            self.change_hostname()
```

Comment:
Can you also change hostname for GCE instances?

Reply:
@bentsi @fruch Should we?

Reply:
I think it's happening automatically already in GCE

Reply:
@amoskong it happens automatically in GCE

Reply:
Thanks.